core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3506,6 +3546,10 @@ private static String javadocLink(String fullyQualifiedClassname) {
     //
     public static final String USER_BLOCK_MASTER_CLIENT_THREADS =
         "alluxio.user.block.master.client.threads";
```

Comment:
how do the new configs relate to this parameter?

Reply:
this parameter is the max pool size. the new min parameter should be less than this. i'll create  an alias so that it is more clear